src/schema/artworkAttributionClasses.ts

```diff
@@ -0,0 +1,12 @@
+import { GraphQLList, GraphQLFieldConfig } from "graphql"
+import { ResolverContext } from "types/graphql"
+import AttributionClass from "./artwork/attributionClass"
+import attributionClasses from "lib/attributionClasses"
+
+const ArtworkAttributionClasses: GraphQLFieldConfig<void, ResolverContext> = {
+  type: new GraphQLList(AttributionClass),
+  description: "List of all artwork attribution classes",
+  resolve: (_root, _args, _context) => Object.values(attributionClasses),
```

Comment:
If we're doing the extra work to expose this, we should probably cast the snake case as camelcase (given that's what we prefer in our graphQL schemas). So this would expose `shortDescription` and `longDescription`.